### PR TITLE
ID-2977 [FEAT] Implement time range form field

### PR DESCRIPTION
**Pull request by armine-fliplet**

### CODE REVIEW

File: js/components/timeRange.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+Fliplet.FormBuilder.field('timeRange', {
+  name: 'Time range',
+  category: 'Text inputs',
```
Comment (tonytlwu) [inline js/components/timeRange.js:3]: Assign this to the **Date & time** category

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/libs/core.js [hunk 2]
```diff
@@ -155,6 +155,7 @@ Fliplet.FormBuilder = (function() {
         });
       };
 
+
```
Comment (tonytlwu) [inline js/libs/core.js:158]:
```suggestion
```

File: templates/configurations/timeRange.interface.hbs [hunk 3]
```diff
@@ -41,5 +47,12 @@
 <div v-else class="form-group">
   <label>Default value</label>
   <div class="text-danger" v-if="errors.requiredField"> \{{ errors.requiredField }} </div>
-  <input type="time" class="form-control basic-time-picker" v-model.trim="value" />
+  <div class="fl-time-range form-html">
```
Comment (tonytlwu) [inline templates/configurations/timeRange.interface.hbs:50]: This doesn't look right to me. https://github.com/Fliplet/fliplet-api/pull/5725/files#diff-4612bda789533f214313c060a4fa54e08258786d64a0f1355e6bcb7a64d96129R13-R29 says a different markup is needed. Can you also share a screenshot of this one it's correctly done?

Comment (armine-fliplet) [reply templates/configurations/timeRange.interface.hbs:50]: image

this is for the custom option

Comment (tonytlwu) [reply templates/configurations/timeRange.interface.hbs:50]: @armine-fliplet Then let's use the same UI markup you've defined for `Fliplet.UI.TimeRange`. It'll ensure consistent look and feel with the time range field as well.

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**